### ISSUE TITLE
Cordova Looking For   res/xml/config.xml   in Wrong Location

### ISSUE THREAD

**Issue by superjoefly** [quasarframework/quasar#1356]
### Software version

Quasar: "quasar-framework": "^0.14.4"
OS: Ubuntu 16.04.3 LTS
Node: v8.9.3
NPM: 5.6.0
Browsers: Chrome
iOS:
Android:
Any other software related to your bug:

Cordova: 8.0.0
Gradle: 4.4.1

### What did you get as the error?
ERROR: Parsing /test/cordova/platforms/android/res/xml/config.xml failed

### What were you expecting?
To be able to install Crosswalk and run "cordova build" command without error

### What steps did you take, to get the error?

// Set up default project:
quasar init default test
cd /test
npm install
quasar dev // success

// Build for production and wrap with Cordova:
cd /test
quasar build
quasar wrap cordova
// does not request to add Crosswalk

// Add Android Platform:
cd /cordova
cordova platform add android

Check Cordova Requirements:
cordova requirements // passes

// Add Crosswalk Plugin:
cordova plugin add cordova-plugin-crosswalk-webview
ERROR: Parsing /test/cordova/platforms/android/res/xml/config.xml failed

// res directory does NOT exist in the above location...

// res directory DOES exist in /home/joey/webdev/LEARNING/Quasar/test/cordova/platforms/android/app/src/main/xml/config.xml

I've also had this problem when trying to run the following build command:

// Build for Release:
cd /test/cordova
cordova build android --release

Error:
Parsing /test/cordova/platforms/android/res/xml/config.xml failed

//////////////////
So it seems Cordova is looking for /res/xml/config.xml in the wrong location.

The reason I'm posting this here is because performing a regular Cordova build succeeds without error.

**Comment by rstoenescu** [comment]
This is 99% Cordova at fault here. Please contact them to get this sorted out. To avoid bouncing back and forth between Quasar and Cordova, write them stepts to reproduce this using Cordova CLI only: cordova create ... then cordova platform add ... etc etc. Thank you.

**Comment by asangster** [comment]
@superjoefly Any word from cordova on this? Cheers

**Comment by superjoefly** [comment]
This was Cordova's response:

"Not a Cordova problem.
Cordova Android 7 changed the project structure. 
That plugin uses hooks that search and write in the old location, so the plugin needs an update.
I think the plugin is not maintained anymore, so you will need to do it yourself and use your fork."

I contacted the Crosswalk plugin maintainer on Github and they created a fork of the plugin that has the correct references for files in Cordova 8: [crosswalk] (https://github.com/kaidokert/cordova-plugin-crosswalk-webview)

However, I found the best solution was to downgrade to Cordova 7.0.1...that seemed to fix all the problems I was having :-)
